src/components/UsageBar.js

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+import UsageBarItem from "./UsageBarItem";
+import {
+  RANK_COMMANDER,
+  RANK_OPERATIVE,
+  RANK_CORPS,
+  RANK_SPECIAL_FORCES,
+  RANK_HEAVY,
+  RANK_SUPPORT
+} from "../util/constants";
+
+const UsageBarBody = styled.div`
+  grid-column: 1 / span 2;
+  border-style: solid;
+  border-width: 1px;
+  border-bottom: 0;
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+`;
+
+const UsageBar = ({
+  points,
+  commanderSlots,
+  operativeSlots,
+  corpsSlots,
+  specialForcesSlots,
+  supportSlots,
+  heavySlots
+}) => (
+  <UsageBarBody>
+    <UsageBarItem name={"Points"} currentValue={points} maxValue={800} /> {/*TODO: Points may need diff styling/ comp*/}
+    <UsageBarItem
+      name={RANK_COMMANDER}
+      currentValue={commanderSlots}
+      maxValue={2}
+    />
+    <UsageBarItem
+      name={RANK_OPERATIVE}
+      currentValue={operativeSlots}
+      maxValue={2}
+    />
+    <UsageBarItem
+      name={RANK_SPECIAL_FORCES}
+      currentValue={corpsSlots}
+      maxValue={3}
+    />
+    <UsageBarItem
+      name={RANK_CORPS}
+      currentValue={specialForcesSlots}
+      maxValue={6}
+    />
+    <UsageBarItem
+      name={RANK_SUPPORT}
+      currentValue={supportSlots}
+      maxValue={3}
+    />
+    <UsageBarItem name={RANK_HEAVY} currentValue={heavySlots} maxValue={2} />
```

Comment:
This one got left out of the multiline club 😢

Reply:
I know, not sure why prettier hates it 😭

Reply:
What error does it spit out? I should start using prettier 💇‍♂

Reply:
No error just when i formatted the whole file that line was untouched haha, maybe due to text length?